espnet/nets/asr_interface.py

```diff
@@ -0,0 +1,55 @@
+class ASRInterface(object):
+    """ASR Interface for ESPnet model implementation
+
+    For chainer backend, __call__ method is needed.
```

Comment:
Please modify this document.
Now chainer also uses the` forward` method, right?

Reply:
Right.